skfp/preprocessing/filters/rule_of_veber.py

```diff
@@ -0,0 +1,82 @@
+from typing import Union
+
+from rdkit.Chem import Mol, rdMolDescriptors
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class RuleOfVeber(BaseFilter):
+    """
+    Rule of Veber
+
+    Rule designed to look for molecules that are likely to exhibit
+    good oral bioavailability. Described in [1]_.
+
+    Molecule must fulfill conditions:
+
+    - number of rotatable bonds >= 10
+    - polar surface area (PSA) >= 140
```

Comment:
topological polar surface area (TPSA)

Reply:
Also, aren't the two rules the other way around? At most 10 rot bonds and TPSA < 140?